java-frontend/src/main/java/org/sonar/java/se/checks/MapComputeIfAbsentOrPresentCheck.java

```diff
@@ -102,20 +103,28 @@ public ProgramState checkPreStatement(CheckerContext context, Tree syntaxNode) {
 
         SymbolicValue keySV = ps.peekValue(1);
         SymbolicValue mapSV = ps.peekValue(2);
-        mapGetInvocations.get(mapSV).stream()
-          .filter(getOnSameMap -> getOnSameMap.withSameKey(keySV))
-          .findAny()
-          .ifPresent(getOnSameMap -> {
-            ObjectConstraint constraint = ps.getConstraint(getOnSameMap.value, ObjectConstraint.class);
-            if (constraint != null && isInsideIfStatementWithNullCheckWithoutElse(mit)) {
-              checkIssues.add(new CheckIssue(context.getNode(), getOnSameMap.mit, mit, getOnSameMap.value, constraint));
-            }
-          });
+        if(!isMethodInvocationThrowingCheckedException(mit.arguments().get(1))) {
```

Comment:
I would prefer to merge the `if` with the one on the line 100 , logically it's equivalent and I believe also more clear